.github/workflows/release.yml

```diff
@@ -1,5 +1,5 @@
-# This workflow should be triggered when a release of the main is needed. 
-# It creates a tag and regenerates the documentation to generate a release. 
+# This workflow should be triggered when a release of the main is needed.
+# It creates a tag and regenerates the documentation to generate a release.
 # It does not bump the version number, that happens for the merged pull request.
```

Comment:
but it does also bump the number right? you even need the "bump-up-version" job

Reply:
correct i don't know what that comment line is doing there haha 
will remove